plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -482,6 +491,9 @@ def _extract_rpm_data(type_id, rpm_filename):
     """
     Extract a dict of information for a given RPM or SRPM.
 
+    !DEPRECATED In favour of using:
+        pulp_rpm.plugins.importers.yum.repmod.primary.process_package_element
```

Comment:
@daviddavis I found it's not so I removed it along with `_update_provides_requires`